### ISSUE TITLE
bq2geojson workflow needs to be refined

### ISSUE THREAD

**Issue by JohnTigue** [codeforseattle/seanetmap#25]
1. We are going to want bq2geojson to be scheduled to build new maps at least daily. The notes in [bq2geojson](https://github.com/m-lab/bq2geojson) imply this can take a chunk of time. So get some experience estimating that.

2. Many of the maps (hexgrid, districts, blocks, tracts) all could benefit from TopoJSON-ification. I would guess that for the hexgrid maps that would be about 5:1 "compression."

3. Are we going to animate the monthly maps? Can we do weekly (currently it only does monthly)?

4. Exactly which maps are we going to feed to it. We need to procure those maps.

5. Will we get into writing our own bq queries, or will we use telescope?

**Comment by JohnTigue** [comment]
See also:
- #24 
- #14 
- #

**Comment by nkinkade** [comment]
Regarding bullet 5 in the original post to this issue, Telescope does not currently support the necessarily functionality for this, and most likely never will.  There is current development in Telescope to allow you to specify additional fields, but I believe it will always force you to select various ISP combinations.

**Comment by JohnTigue** [comment]
Regarding point 3 in the original post to the issue:
Might be that we can query any time period we like:
>The above case is assuming that people want to retrieve data by months. However, hours, minutes and seconds could be specified to modify the time range. Since Big Query allows Unix timestamps, you could query specifying a range of Unix timestamps. For example, if we need data from 00:00:00, Jan 20, 2013 to 16:00:00, March 02, 2013, the starting time could be converted to 1358640000, and it ends at 1362240000.

https://docs.google.com/document/d/1Y97QwbSIHGN_Q6BaWD5z-Zh_I0uKe95S_9jkDh3fWYY/edit#

**Comment by JohnTigue** [comment]
@nkinkade has landed the TopoJSON work. Nice!

**Comment by JohnTigue** [comment]
AWS Lambda can be scheduled. See the Architectural overview doc for more info.

**Comment by nkinkade** [comment]
A quick comment on point 1 in the original post.  My comment about bq2geojson perhaps taking a really long time to run was mostly based on hexgrids using a very small cell width over a very large area.  Processing for each month completes in just a few seconds using the City Council districts polygon file and Big Query data limited to just the Seattle area.

**Comment by JohnTigue** [comment]
Agreed, I am seeing the same. I just like to document everything other wise I forget.
